### ISSUE TITLE
Code for Train.py and Shallow Discriminator Network

### ISSUE THREAD

**Issue by srinadhu** [jia-wan/ResidualRegression-pytorch#1]
Hello @jia-wan ,

Thanks for making the code public. If possible can you provide the Train file as well i.e. train.py? The paper mentions adversarial loss which is totally clear, it also mentions the use of a shallow discriminator it's not clear. The shallow discriminator is not in the model.py as well. Where is the discriminator loss going i.e., the discriminator architecture is classifying predicted and actual, that loss?. Thank you.

**Comment by hellowangqian** [comment]
Like to see the train file as well, hope the author can release the whole stuff to the community, much appreciated.
